[Doc Update][Extend a Teams personal tab across Microsoft 365] (#7596)
MicrosoftDocs/msteams-docs: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

If you have an existing personal tab app, make a copy or a branch of your production project for testing and update your App ID in the app manifest to use a new identifier (distinct from the production App ID, for testing).

> If you'd like to use sample code to complete this tutorial, follow the setup steps in the [Todo List Sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/main/todo-list-with-Azure-backend) to build a personal tab app using the Teams Toolkit extension for Visual Studio Code, then return to this article to update it for Microsoft 365. You can also use the Teams Toolkit extension to create a new Teams app and select *Start from a sample*. One of the options available is the *Todo List with backend on Azure* sample.

Reviewer: ```suggestion
If you'd like to use sample code to build a personal tab app using the Teams Toolkit extension for Visual Studio Code, follow the setup steps in the [Todo List Sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/main/todo-list-with-Azure-backend). After you've created a personal tab, return to this article to extend your personal tab across Microsoft 365. You can also use the Teams Toolkit extension to create a new Teams app and select *Start from a sample*. 
```

Author: This is more about using sample code vs your own production code. I've revised this section in the latest commit.

Alternately, you can use a basic single sign-on *hello world* app already enabled for Microsoft 365 in the following [Quickstart](#quickstart) section and then skip to [sideload your app in Teams](#sideload-your-app-in-teams).

### Quickstart

> To start with a [personal tab](https://github.com/OfficeDev/TeamsFx-Samples/tree/ga/todo-list-with-Azure-backend-M365) that's already enabled to run in Outlook and Office, you can use the Teams Toolkit extension for Visual Studio Code.

Reviewer: The link to personal tab here redirects to Azure-backend-M365. The Quickstart guide has the steps for single sign-on hello world app.

Request you to update the link to single sign-on hello world app.

Author: Noted and removed link in the latest commit.
